code/game/gamemodes/wizard/artefact.dm

```diff
@@ -836,6 +836,8 @@ GLOBAL_LIST_EMPTY(multiverse)
 				var/wgw =  sanitize(input(user, "What would you like the victim to say", "Voodoo", null)  as text)
 				target.say(wgw)
 				log_game("[user][user.key] made [target][target.key] say [wgw] with a voodoo doll.")
+				log_say("Wicker doll say to [target][target.key]: [wgw]", src)
+				log_admin("[user][user.key] made [target][target.key] say [wgw] with a voodoo doll.")
```

Comment:
Can you add a `user.create_log(SAY_LOG, "said something through [src] "[something].....")`. This will give the logging system ingame an entry as well

Reply:
Will do it tomorrow